style.css

```diff
@@ -1,8 +1,167 @@
 #app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   text-align: center;
-  color: #2c3e50;
+  color: #000;
   margin-top: 60px;
 }
+.container {
+  max-width: 1200px;
+  width: 100%;
+  margin: 0 auto;
+}
+h1, h2, h3, h4, h5 {
+  font-family: 'Roboto Mono', monospace;
+  color: #000;
+}
+.main-wrapper h1 {
+  margin: 0 0 50px;
+  font-size: 48px;
+  font-weight: 400;
+}
+.main-wrapper h1, .main-wrapper .search-bar-wrapper {
+  text-align: center;
+}
+.main-wrapper .search-bar-wrapper {
+  margin: 0 auto 70px;
+  max-width: 300px;
+  width: 100%;
+  position: relative;
+}
+.main-wrapper .search-bar-wrapper input {
+  padding: 10px;
+  font-size: 16px;
+  border: 1px solid #000;
+  border-radius: 0;
+  width: 100%;
+  height: 100%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  background-image: url('images/search-icon.svg');
+  background-size: 20px;
+  background-position: calc(100% - 10px) center;
+  background-repeat: no-repeat;
+}
+.main-wrapper .search-bar-wrapper input:focus::placeholder {
+  color: transparent;
+}
+.main-wrapper .search-bar-wrapper input:focus {
+  border: 1px solid #000;
+  outline: none;
+}
+.no-results-wrapper p {
+  color: #000;
+  font-weight: 600;
+  text-align: center;
+  margin: 100px 0 48px;
+}
+.movie-grid {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 48px 75px;
+}
+.movie-grid .movie-grid-item .movie-grid-inner {
+  position: relative;
+  cursor: pointer;
+}
+
+.movie-grid .movie-grid-item img {
+  width: 350px;
+  max-width: 100%;
+  height: 374px;
+  border-radius: 20px;
+}
+.movie-grid .movie-grid-item .movie-grid-inner:hover .movie-content{
```

Comment:
Please check the specificity of each and every selector and make sure they are under the specificity 0,3,0 even 0,3,1 is greater than 0,3,0.